app/Jobs/RockTheVote/ImportRockTheVoteRecord.php

```diff
@@ -126,7 +126,7 @@ private function formatResponse(NorthstarUser $user, array $post)
             ]),
         ];
 
-        $userFields = ['id', 'email', 'mobile', 'voter_registration_status', 'sms_status', 'sms_subscription_topics', 'email_subscription_status', 'email_subscription_topics', 'referrer_user_id'];
+        $userFields = ['id', 'email_preview', 'mobile_preview', 'voter_registration_status', 'sms_status', 'sms_subscription_topics', 'email_subscription_status', 'email_subscription_topics', 'referrer_user_id'];
```

Comment:
I think we need to make these changes too per https://github.com/DoSomething/chompy/pull/177/commits/76c08f3012b5e97ea52ff76c45606c563fafac81

Reply:
Oh, yep! Done in 1a7a800.

Reply:
Although, I'm realizing now that those fields will not be included in the [`getUserByEmail`](https://github.com/DoSomething/chompy/pull/177/files#diff-cd7de09fd28f032192b6f11e74524a0dR165) or [`getUserByMobile`](https://github.com/DoSomething/chompy/pull/177/files#diff-cd7de09fd28f032192b6f11e74524a0dR178) calls unless we specifically ask for them (and so we'd have to either request them across the board, or perhaps pass another `$debug` parameter to the job to tell it whether or not to query those...)

Reply:
I'm wondering if printing the provided form values (from the `$record` passed to this job), and non-sensitive results (just a boolean "found email user" or "found mobile user" or the IDs for each) would provide us enough context for debugging?

Reply:
I think printing the form values should work! That form submission flash message currently doesn't indicate whether user/post was updated/created (would be sweet if it did -- but I'd imagine that's just a nice-to-have since we don't do that)

Reply:
I don't think it's a biggie to lose pre-populating the form with current user info (it's convenient to have the email ready to be edited, but also very easy to clear out your mobile - which prevents creating a new user)